inc/class-s3-uploads-stream-wrapper.php

```diff
@@ -2,16 +2,23 @@
 
 class S3_Uploads_Stream_Wrapper extends Aws\S3\StreamWrapper {
 
+	protected static $s3_uploads;
 	/**
 	 * Register the 's3://' stream wrapper
 	 *
 	 * @param S3Client $client Client to use with the stream wrapper
 	 */
-	public static function register( Aws\S3\S3Client $client) {
-		stream_wrapper_register( 's3', __CLASS__, STREAM_IS_URL );
-		static::$client = $client;
-	}
+	public static function register_streamwrapper( $s3_uploads )
+	{	
```

Comment:
Watch the coding standards here; WP coding standards dictate the brace on the same line as the declaration :)